src/main/java/seedu/address/logic/commands/schedule/ScheduleCommand.java

```diff
@@ -0,0 +1,31 @@
+package seedu.address.logic.commands.schedule;
+
+import seedu.address.logic.commands.Command;
+
+public abstract class ScheduleCommand extends Command {
+    public static final String COMMAND_WORD = "schedule";
+
+    public static final String MESSAGE_USAGE = "The following formats are supported for " + COMMAND_WORD + ":\n"
+            + COMMAND_WORD + " add candidate/CANDIDATE_INDEX at/DATE_TIME \n"
+            + COMMAND_WORD + " edit INDEX DATE_TIME \n"
+            + COMMAND_WORD + " delete INDEX";
+
+    public static final String MESSAGE_DUPLICATE_CANDIDATE_INTERVIEW =
+            "Interview for this candidate already exists!";
+
+    public static final String MESSAGE_CONFLICTING_INTERVIEW =
+            "Interview for another candidate clashes with the proposed time slot!";
+
+    public static final String MESSAGE_CANDIDATE_NOT_AVAILABLE =
+            "Candidate is not available on the proposed interview day!";
+
+    public static final String MESSAGE_NOT_OFFICE_HOUR =
```

Comment:
A minor comment, but jotting it down in case it slips my mind. Perhaps the message should also indicate the last interview can be scheduled at 5.30pm daily i.e. `17:50`?

Reply:
^ I'm also realising it was me who typed the original `MESSAGE_NOT_OFFICE_HOUR` sorry 😅

Reply:
Added one line in the latest commit! Not sure if it is properly worded but can always update again 😄